client/components/domains/featured-domain-suggestions/styles/base.scss

```diff
@@ -85,9 +93,45 @@
 	}
 }
 
-.featured-domain-suggestions--has-match-reasons .featured-domain-suggestion {
-	.domain-registration-suggestion__match-reasons {
-		order: 3;
-		flex-basis: 26px;
+@include breakpoint( '<660px' ) {
+	.featured-domain-suggestions {
+		flex-flow: wrap;
+
+		.featured-domain-suggestion .domain-suggestion__action.button.is-primary {
+			width: initial;
+		}
+	}
+}
+
+@include breakpoint( '>660px' ) {
+	.featured-domain-suggestions {
+		.domain-registration-suggestion__title {
+			.featured-domain-suggestions--title-in-18em & {
```

Comment:
I might be missing something, but I don't see where these are still used. If they aren't used, can we remove them?

Reply:
Hi Del, this is being used [here](https://github.com/Automattic/wp-calypso/blob/692f4a15ad116f5850739d436f54118a572fedcd/client/components/domains/featured-domain-suggestions/index.jsx#L49) in order to dynamically reduce the font size according to the domain name length.